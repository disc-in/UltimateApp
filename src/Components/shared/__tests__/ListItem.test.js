import React from 'react';
import { render } from '@testing-library/react-native';
import { Text } from 'react-native';

import ListItem from '../ListItem';

describe('<ListItem />', () => {
  it('renders correctly', () => {
    const { toJSON } = render(
      <ListItem>
        <Text>Hello world</Text>
      </ListItem>,
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
