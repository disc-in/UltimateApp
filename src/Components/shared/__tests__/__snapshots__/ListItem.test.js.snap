// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "elevation": 3,
      "marginTop": 10,
      "padding": 10,
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.3,
      "shadowRadius": 3,
    }
  }
>
  <Text>
    Hello world
  </Text>
</View>
`;
